target/linux/ar71xx/image/nand.mk

```diff
@@ -57,6 +57,18 @@ define Device/hiveap-121
 endef
 TARGET_DEVICES += hiveap-121
 
+#define Device/GL-AR300MD
+  #DEVICE_TITLE := GL AR300M(Dual)
+  #DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-storage kmod-usb-ledtrig-usbport kmod-ath10k ath10k-firmware-qca988x
```

Comment:
According to GL.iNet it's only single band so you should only need ath9k, GL-AR300MD is dual band however.
https://www.gl-inet.com/product/gl-ar300m/